lib/octokit/client/gists.rb

```diff
@@ -45,10 +45,16 @@ def starred_gists(options = {})
       # Get a single gist
       #
       # @param gist [String] ID of gist to fetch
+      # @option options [String] :sha Specific gist revision SHA
       # @return [Sawyer::Resource] Gist information
       # @see https://developer.github.com/v3/gists/#get-a-single-gist
+      # @see https://developer.github.com/v3/gists/#get-a-specific-revision-of-a-gist
       def gist(gist, options = {})
-        get "gists/#{Gist.new(gist)}", options
+        if sha = options.delete(:sha)
```

Comment:
Good question. Since [the docs](https://developer.github.com/v3/gists/#get-a-specific-revision-of-a-gist) also use `:sha` I guess it's okay to use that here too.